### PR TITLE
Feat/registration

### PR DESCRIPTION
**Pull request by ninadarbaidze**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,32 +1,65 @@
 {
-  "name": "movie-quotes-api",
+  "name": "inside-joke---backend",
```
Comment (giunashvili) [inline package.json]: what insidejoke?

File: src/controllers/auth-controller.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    res.status(201).json({
+      message: 'User Created Successfully',
+      
+    })
+
+    const token = jwt.sign(
+      {
+        userId: response._id,
+      },
+      process.env.JWT_SEC!,
```
Comment (giunashvili) [inline src/controllers/auth-controller.ts]: you have ! at the end of env variable:
process.env.JWT_SEC!

File: src/views/email.handlebars [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (giunashvili) [inline src/views/email.handlebars:1]: prettify a little bit

**Review by giunashvili — CHANGES_REQUESTED** [review]

File: src/views/email.handlebars (hunk 3 above)
Comment (ninadarbaidze) [reply src/views/email.handlebars:1]: should I make the email template prettier? or code itself

Comment (giunashvili) [reply src/views/email.handlebars:1]: the code. like indentation and stuff like that

**Review by giunashvili — CHANGES_REQUESTED** [review]
Use environment.d.ts file to define env variables for typescript to pick up those vars

**Review by giunashvili — APPROVED** [review]

**Pull request merged**